examples/trading/frontend/README.md

```diff
@@ -0,0 +1,57 @@
+# Trading e2e demo - Frontend
+
+This dApp was created using `@mysten/create-dapp` that sets up a simple React
```

Comment:
```suggestion
This dApp was created using `@mysten/create-dapp` that sets up a basic React
```
Same note on "simple" as "easy" :)

Reply:
(I believe this bit is generated by the `create-dapp` script so we will also need to update the wording at the source).

Reply:
Indeed, fixed it from all places I could find it!